log_analyzer/config.json

```diff
@@ -1,25 +1,110 @@
 {
+    "port": 8000,
+    "ip": "127.0.0.1",
+    "packets_cycle_threshold": 1000,
+    "pipeline_measurements": [
+        {
+            "name": "total_execution_time_ratio_measurement",
```

Comment:
From just the names, it is not completely clear what ratio we are talking about. A ratio is something / something else.

Reply:
@baselqyqony if you can find a good name which describes that would be a great help.

Reply:
@baselqyqony it will also be helpful to add the explanation for the checkers and measurements since I guess you know better from this side.